core/src/main/java/com/novoda/merlin/MerlinBuilder.java

```diff
@@ -77,13 +78,23 @@ public MerlinBuilder withAllCallbacks() {
     }
 
     /**
-     * Sets the internal Merlin Logging state. Uses the TAG : Merlin
+     * Sets the internal Merlin logging state to report to a given LogHandle.
      *
-     * @param withLogging by default Logging is disabled. withLogging = true will enable logging, withLogging = false will disable.
+     * @param logHandle that should be used when logging.
      * @return MerlinBuilder.
      */
-    public MerlinBuilder withLogging(boolean withLogging) {
-        MerlinLog.LOGGING = withLogging;
+    public MerlinBuilder withLogging(Logger.LogHandle logHandle) {
```

Comment:
> true perhaps attach a default logHandle and the false do nothing?

I was thinking this too but false would need to clear the `Logger` because it has handles 😕